js/models/contact_model.js

```diff
@@ -307,6 +309,7 @@ angular.module('contactsApp')
 				return matchingProps.length > 0;
 			},
 
+			/* eslint-disable no-console */
```

Comment:
This, the two code comments (L321, L327) and the `console.debug` calls (L351, L356) should be removed.